.docker/Dockerfile.cli

```diff
@@ -1,16 +1,33 @@
-FROM govcms8lagoon/govcms8
+##
+# `cli` image.
+#
+# The application is built in this image as it contains build tools like
+# composer and npm.
+#
+# The default PaaS build strategy is:
+#
+#  * wipe the /app directory, from the upstream image build (which is currently SaaS oriented).
+#  * prepare and run composer install inside the `cli` container
+#  * copy in the non-composer files
+#
 
+FROM amazeeio/php:7.2-cli-drupal
+
+# Clean out anything that might be in /app directory and set up the initial files.
 RUN rm -rf /app
-COPY composer.json composer.lock /app/
-COPY scripts /app/scripts
```

Comment:
By running the COPY first (noting the new .dockerignore) these are redundant COPYs.

Reply:
@simesy no, they are not redundant. this is how Docker caching works. we are doing these in order to cache composer dependencies. please do not remove this